mmcls/core/evaluation/multilabel_eval_metrics.py

```diff
@@ -0,0 +1,68 @@
+import warnings
+
+import numpy as np
+import torch
+
+
+def average_performance(pred, target, thrs=None, k=None):
+    """Calculate CP, CR, CF1, OP, OR, OF1, where C stands for per-class
+        average, O stands for overall average, P stands for precision, R
+        stands for recall and F1 stands for F1-score
+
+    Args:
+        pred (torch.Tensor | np.ndarray): The model prediction with shape
+            (N, C), where C is the number of classes.
+        target (torch.Tensor | np.ndarray): The target of each prediction with
+            shape (N, C), where C is the number of classes. 1 stands for
+            positive examples, 0 stands for negative examples and -1 stands for
+            difficult examples.
+        thrs (float): The confidence threshold. Defaults to None.
+        k (int): Top-k performance. Note that if thrs and k are both given, k
+            will be ignored. Defaults to None.
+
+    Returns:
+        tuple: (CP, CR, CF1, OP, OR, OF1)
+    """
+    if isinstance(pred, torch.Tensor) and isinstance(target, torch.Tensor):
+        pred = pred.numpy()
+        target = target.numpy()
+    elif not (isinstance(pred, np.ndarray) and isinstance(target, np.ndarray)):
+        raise TypeError('pred and target should both be torch.Tensor or'
+                        'np.ndarray')
+    if thrs is None and k is None:
+        thrs = 0.5
```

Comment:
check the default value according to Head implementations.

Reply:
I will put sigmoid in simple test in Head implementation, therefore, perhaps we can leave default thr as 0.5 here?